staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -237,7 +223,7 @@ func (t *tracker) Get(gvk schema.GroupVersionKind, ns, name string) (runtime.Obj
 
 	if status, ok := obj.(*metav1.Status); ok {
 		if status.Details != nil {
-			status.Details.Kind = gvk.Kind
+			status.Details.Kind = gvr.Resource
```

Comment:
This looks like dead code. I removed the `if` block and no tests were broken. I'll just remove it.